bb8/src/inner.rs

```diff
@@ -146,6 +160,14 @@ where
         }
     }
 
+    /// Returns statistics about the historical usage of the pool.
+    pub(crate) fn statistics(&self) -> Statistics {
+        let gets = self.pool_inner_stats.gets.load(Ordering::SeqCst);
```

Comment:
Would prefer to isolate this into an `impl From<AtomicStatistics> for Statistics`.